website/screens/components/select/specs/SelectSpecsPage.tsx

```diff
@@ -143,1056 +350,842 @@ const sections = [
+              <thead>
+                <tr>
+                  <th>Property</th>
+                  <th>Element</th>
+                  <th>Token</th>
+                  <th>Value</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>
+                    <Code>padding-left</Code>
+                  </td>
+                  <td>List dialog</td>
+                  <td>
+                    <Code>spacing-8</Code>
+                  </td>
+                  <td>0.5rem / 8px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>padding-right</Code>
+                  </td>
+                  <td>List dialog</td>
+                  <td>
+                    <Code>spacing-8</Code>
+                  </td>
+                  <td>0.5rem / 8px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>margin-top</Code>
+                  </td>
+                  <td>List dialog content</td>
+                  <td>
+                    <Code>spacing-4</Code>
+                  </td>
+                  <td>0.25rem / 4px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>margin-bottom</Code>
+                  </td>
+                  <td>List dialog content</td>
+                  <td>
+                    <Code>spacing-4</Code>
+                  </td>
+                  <td>0.25rem / 4px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>padding-top</Code>
+                  </td>
+                  <td>List option</td>
+                  <td>
+                    <Code>spacing-4</Code>
+                  </td>
+                  <td>0.25rem / 4px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>padding-bottom</Code>
+                  </td>
+                  <td>List option</td>
+                  <td>
+                    <Code>spacing-4</Code>
+                  </td>
+                  <td>0.25rem / 4px</td>
+                </tr>
+              </tbody>
+            </DxcTable>
+          </>
+        ),
+      },
+      {
+        title: "Size",
+        content: (
+          <>
+            <DxcParagraph>
+              The component <Code>width</Code> can adopt the following values:
+            </DxcParagraph>
+            <DxcTable>
+              <thead>
+                <tr>
+                  <th>Width</th>
+                  <th>Value</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>
+                    <Code>small</Code>
+                  </td>
+                  <td>240px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>medium</Code> (<em>default</em>)
+                  </td>
+                  <td>360px</td>
```

Comment:
The same here for `default`. I would not add it here, to keep it as the rest of components. Also I think the place of this default explanation is the code (in the props table, as it is now).

Reply:
Maybe this was done by Alex to let the user know that the Specifications examples have a `"medium"` size.

Reply:
Regarding the blank space, there is already one, I don't fully understand this comment 🤔.

Reply:
I've only seen this `default` explanation in `Select` and `Text Input`. In the rest of components,  it is not specified the default. I think we should keep it like this also in Select and Text input, since in any other specifications prop the default one is explained, and it makes sense to me. The default values should only be explained in the props table in code tab.

Which blank space do you refer here? I've only commented about the blank space in the comment above:
![image](https://user-images.githubusercontent.com/10975289/196118186-f3a426b1-dd3c-4aa5-951d-2912bb17b971.png)

Reply:
It is not only present in the Select and Text Input, take a look:

**Dialog**:
![image](https://user-images.githubusercontent.com/44321109/196222320-0a7d1a44-637e-4518-824a-7d07baec97ad.png)

**Dropdown**:
![image](https://user-images.githubusercontent.com/44321109/196222514-ef87de15-62e6-497e-b115-6b3144b09fba.png)

Shall we also remove it in these places?

Reply:
Yes, I would remove the `(default)` from `Dialog` and `Dropdown` too, since it is not present in `Margin`, `Padding`, and `Width` sections of any of the rest of the components.